### PR TITLE
[Doc repo] Remove Delete column if user doesn't have delete permissions

### PR DESCRIPTION
**Pull request by zaliqarosli**
Redmine: https://redmine.cbrain.mcgill.ca/issues/11251

Apologies in advance if the code is not great - trying to understand how Mustache.js communicates with the other languages was a struggle. Open to all reviews/suggestions! Thanks :)

### CODE REVIEW

File: modules/document_repository/js/document_repository_helper.js [hunk 1]
```diff
@@ -195,6 +195,7 @@ function uploadForm() {
 function renderTree() {
   var fileDir = JSON.parse($("#json_data").html());
   var filtered = JSON.parse($("#isFiltered").html()).filtered;
+  var hasDeletePerm = JSON.parse($("#hasDeletePerm").html()).hasDeletePerm;
```
Comment (davidblader) [inline modules/document_repository/js/document_repository_helper.js]: this seems like a bad idea. can you just use `loris.userHasPermission('document_repository_delete')` ?

Comment (zaliqarosli) [reply modules/document_repository/js/document_repository_helper.js]: did not know we could do that is js!

File: modules/document_repository/js/document_repository_helper.js [hunk 2]
```diff
@@ -239,6 +240,13 @@ function renderTree() {
         //new table layout
         var file = $('#file').html();
         Mustache.parse(file);   // optional, speeds up future uses
+
+        //pass delete permissions info
```
[the remark this answers is not in the document]
Comment (zaliqarosli) [reply modules/document_repository/js/document_repository_helper.js]: done

**Comment by zaliqarosli** [comment]
@xlecours @davidblader thanks for the comments, ready for re-review and re-test @PapillonMcGill

**Comment by HenriRabalais** [comment]
Tested again and passed manual test!

**Pull request merged**